core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -2337,6 +3310,55 @@ public void testGetAllStoreAllMatchCacheAccessExceptionBeforeLoaderSomeMatchDisj
    * <ul>
    *    <li>non-empty request key set</li> 
    *    <li>all {@link Store} entries match</li>
+   *    <li>{@link Store#bulkComputeIfAbsent} throws before accessing loader</li>
+   *    <li>some {@link CacheLoaderWriter} entries match</li>
+   *    <li>non-matching {@link CacheLoaderWriter#loadAll(Iterable)} calls fail witb {@link BulkCacheLoadingexception}</li>
```

Comment:
Two typos on this line:
* witb -> with
* BulkCacheLoadingexception -> `BulkCacheLoadingException`